src/docs/development/platform-integration/platform-views.md

```diff
@@ -444,13 +444,13 @@ import Flutter
 import UIKit
 
 class FLNativeViewFactory: NSObject, FlutterPlatformViewFactory {
-    private weak var messenger: FlutterBinaryMessenger
-
+    private var messenger: FlutterBinaryMessenger
+    
```

Comment:
nit: remove this and the trailing whitespaces below.

Reply:
Trailing whitespaces removed